exchanges/suspense/package.json

```diff
@@ -19,8 +19,8 @@
     "react",
     "suspense"
   ],
-  "main": "dist/urql-exchange-suspense",
-  "module": "dist/urql-exchange-suspense.mjs",
+  "main": "dist/urql-exchange-suspense.js",
+  "module": "dist/urql-exchange-suspense.es.js",
```

Comment:
I think we’ll also want “exports” removed. I’ll do another check and maybe a subsequent PR as I’m unsure whether we need the dependency on React or whether it’s just to indicate that it’s _for_ React 😅